.gitignore

```diff
@@ -3,3 +3,4 @@ npm-debug.log
 es
 lib
 node_modules
+.clinic
```

Comment:
We'd rather not have non-package-related exclusions checked into source control. Directories and files can be ignored locally (and not checked into source control) by moving this line to `.git/info/exclude`. This command will do it for you: `echo .clinic >> .git/info/exclude`

Reply:
Roger. Fixed!